platform/ios/src/MGLMapView.mm

```diff
@@ -1070,6 +1070,8 @@ - (void)sleepGL:(__unused NSNotification *)notification
     {
         self.dormant = YES;
 
+        [MGLOfflineStorage sharedOfflineStorage].mbglFileSource->setRevalidationState(mbgl::DefaultFileSourceRevalidationState::Inactive);
```

Comment:
That would also allow iOS and macOS to share the same implementation. For example, the iOS implementation of MGLMapView calls `-sleepGL:` in response to `UIApplicationDidEnterBackgroundNotification`, which probably corresponds to `NSApplicationDidHideNotification` and/or `NSApplicationDidChangeOcclusionStateNotification` on the Mac.